www/templates/html/SiteMaster/Core/Registry/Site/AddSiteForm.tpl.php

```diff
@@ -1,8 +1,24 @@
 <form action="<?php echo $context->getEditURL(); ?>" method="POST">
     <ul>
         <li>
-            <label for="base_url"><span class="required">(required)</span> The base URL of the site</label>
-            <input type="url" id="base_url" name="base_url" placeholder="http://www.yoursite.edu/" autofocus required />
+            <?php
+            
+            $value = '';
+            if (isset($_POST['base_url'])) {
+                $value = 'value="' . $savvy->escape($_POST['base_url']) . '"';
+            }
+            
+            if (isset($context->options['recommended'])) {
+                $value = 'value="' . urldecode($context->options['recommended']) . '"';
```

Comment:
Good point, I didn't realize that values in the super global arrays are already decoded.